10_code/threshold.py

```diff
@@ -114,23 +114,28 @@
 Texas, deaths from 2003 to 2014, threshold is 2007
 Control states = "TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK",
 """
-group_tx = ["TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK"]
-group_tx_names = ["Texas","Utah","Georgia","Colorado", "California","North Dakota","Illinois","Louisiana","Maryland","Oklahoma"]
+texas_states = ["TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK"]
+texas_names = ["Texas","Utah","Georgia","Colorado", "California","North Dakota","Illinois","Louisiana","Maryland","Oklahoma"]
 
-deaths_tx = overdose_filtered_no_missing[(overdose_filtered_no_missing["State"].isin(group_tx))]
+
+deaths_tx = overdose_filtered_no_missing[(overdose_filtered_no_missing["State"].isin(texas_states))]
```

Comment:
This subset can also drop the 2015 data since we wont need it moving further

Reply:
Yes, I later dropped 2015. Please refer line  194 in "overdose_deaths_tx"